CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
-## How to contribute to the this project
+# How to contribute to this project
 
-Please use our tracker for new relation requests, etc
+Please use our tracker for requests for new relations, modifications, clarifications, etc.
 
 https://github.com/oborel/obo-relations
 
-## Contributing via Pull Requests
+Creating a new issue in the tracker allows the RO editors and community to discuss proposed changes, fixes, or additions. If you'd like faster turnaround on your request, and you know how to use GitHub, please specify your desired changes by submitting a GitHub Pull Request on the appropriate file.
 
-We hope to be able to accept contributions to the ontology in the form of Pull Requests in the future. Note that this is currently difficult for OWL, see:
+## Before you write a new request, please consider the following: 
 
-http://douroucouli.wordpress.com/2014/03/30/the-perils-of-managing-owl-in-a-version-control-system/
+- **Does the term already exist?** Before submitting suggestions for new ontology terms, check whether the term exist, either as a primary term or a synonym term. You can search using [OLS](http://www.ebi.ac.uk/ols/ontologies/ro)
 
-OWLAPI developers are working to circumvent this. In the interim, PRs can only be accepted if you have a LOCK on the editors file. See [src/ontology/README-editors.md](src/ontology/README-editors.md) for details.
+## Guidelines for creating GitHub tickets with contributions to the ontology:
+
+1. **Write a detailed request:** Please be specific and include as many details as necessary, providing background information, and if possible, suggesting a solution. GOC editors will be better equipped to address your suggestions if you offer details regarding *'what is wrong'*, *'why'*, and *'how to fix it'*.
+
+2. **Provide examples and references:** Please include PMIDs for new term requests, and include also screenshots, or URLs illustrating the current ontology structure for other types of requests. 
```

Comment:
Maybe just generalize to "Please include any applicable references" rather than PMIDs. I think RO relations are a little different in this regard compared to GO terms.

Reply:
sounds good, thanks for the suggestions.